middleware/session/session.go

```diff
@@ -171,6 +171,11 @@ func (s *Session) setCookie() {
 
 	s.ctx.Response().Header.SetCookie(fcookie)
 	fasthttp.ReleaseCookie(fcookie)
+
+	if s.config.Source == SourceHeader {
+		s.ctx.Request().Header.SetBytesV(s.config.CookieName, []byte(s.id))
+		s.ctx.Response().Header.SetBytesV(s.config.CookieName, []byte(s.id))
+	}
```

Comment:
A cookie will still be set even if the source is Header. We should put the above code in an if statement to prevent this. We should also rename the function to `setSession` or similar

Reply:
Thanks! it has been adjusted to `setSession`